composer.json

```diff
@@ -68,7 +68,8 @@
         "illuminate/support": "self.version",
         "illuminate/translation": "self.version",
         "illuminate/validation": "self.version",
-        "illuminate/view": "self.version"
+        "illuminate/view": "self.version",
+        "tightenco/collect": "self.version"
```

Comment:
If let say laravel/framework make a release today, and tightenco/collect are only available to run the subsplit/publish command few days/weeks (not saying it would be common, but possible). There wouldn't be any exact version available.

In this case, wouldn't both the version be installed? Or composer smart enough to validate `"laravel/framework":"v5.3.0"` with `"tightenco/collect":"5.3.x-dev"`?